README.md

```diff
@@ -131,6 +131,8 @@ import numpy as np
 frame_data = np.random.randint(0, 100, size=(2**10, 2**8))
 df = pd.DataFrame(frame_data)
 ```
+**In local (without a cluster) modin will create and manage a simulated (dask or ray) cluster for the execution**
```

Comment:
Can we change the word `simulated` to `local`? Simulated is not a term that will be familiar to the readers.

Reply:
Okay